llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -4007,7 +4007,8 @@ SDValue AMDGPUTargetLowering::performIntrinsicWOChainCombine(
   case Intrinsic::amdgcn_rcp_legacy:
   case Intrinsic::amdgcn_rsq_legacy:
   case Intrinsic::amdgcn_rsq_clamp:
-  case Intrinsic::amdgcn_tanh: {
+  case Intrinsic::amdgcn_tanh:
+  case Intrinsic::amdgcn_prng_b32: {
```

Comment:
This is unrelated to base support, the comment is unrelated, and I'm not sure it's safe to fold this

Reply:
It is related to the changes in `llvm/test/CodeGen/AMDGPU/llvm.amdgcn.prng.ll`.